### ISSUE TITLE
custom field filterable is broken

### ISSUE THREAD

**Issue by Grokzen** [ninech/netbox-docker#65]
## Current Behavior

The field `filterable` is not working as expected. Currently it accepts a `true/false` bool but inside netbox it has changed to a PositiveSmallInteger field. No matter what value is set in the custom fields .yml file, it will default to FILTER_LOOSE

CustomField model https://github.com/digitalocean/netbox/blob/develop/netbox/extras/models.py#L86

choices constants defentiion https://github.com/digitalocean/netbox/blob/develop/netbox/extras/constants.py#L33

I am running on Netbox tag 2.3.2

...

## Expected Behavior

This field must be possible to set or to disable. It matters less the choice between loose/exact, but more that i can disable the field for filtering, but that is not possible right now.

## Possible solution

I am not sure how to really change this. I see 2 possible solutions

```
# 1. in the yaml file use one variable to define the filter logic state. It is nice to have one variable, but not clean to mix string/boolean values in the same variable.
...
filter_logic: loose / exact / false
...
# OR
...
filter_logic: loose / exact / disabled
...
```

```
2. In the yaml file use 2 variables, one to controll if the field should be filterable, and one the controlls the logic part.
...
filterable: true / false
filter_logic: loose / exact
```

**Comment by cimnine** [comment]
I would prefer `filter_logic: loose / exact / disabled`.

**Comment by Grokzen** [comment]
I implemented that way solution in the referenced MR.

One question tho @cimnine, should the old solution still be around for legacy reasons so that the script still supports older versions of netbox? or do you have some kind of support level policy regarding that script?

**Comment by thde** [comment]
Closed #66
